### ISSUE TITLE
Maven artifact references missing parent artifact

### ISSUE THREAD

**Issue by LoadingByte** [rabanti-github/NanoXLSX4j#2]
nanoxlsx4j's POM in Maven Central ( https://repo1.maven.org/maven2/ch/rabanti/nanoxlsx4j/2.3.3/nanoxlsx4j-2.3.3.pom ) has a `<parent>` block that references the `nanoxlsx4j-root` artifact. However, that artifact is not available in Central, so Gradle complains:

```
> Could not resolve all artifacts for configuration ':runtimeClasspath'.
   > Could not resolve ch.rabanti:nanoxlsx4j:2.3.3.
     Required by:
         project :
      > Could not resolve ch.rabanti:nanoxlsx4j:2.3.3.
         > Could not parse POM https://repo.maven.apache.org/maven2/ch/rabanti/nanoxlsx4j/2.3.3/nanoxlsx4j-2.3.3.pom
            > Could not find ch.rabanti:nanoxlsx4j-root:2.3.3.
              Searched in the following locations:
                - https://repo.maven.apache.org/maven2/ch/rabanti/nanoxlsx4j-root/2.3.3/nanoxlsx4j-root-2.3.3.pom
```

Would it be possible to either rewrite the uploaded POM to not include the parent, or to also upload the parent POM to Central?

**Comment by rabanti-github** [comment]
Hi,
Thanks for the report. I will have a look into this ASAP

**Comment by LoadingByte** [comment]
Thank you so much for your quick response!

**Comment by rabanti-github** [comment]
Hi,
Technically, there is a new version 2.4.0 that deals with the described problem and also implemented the default column style (issue: https://github.com/rabanti-github/NanoXLSX4j/issues/3).
However, currently, I cannot publish this version since Maven Central has changed the deployment process... and as always, they do their best to make it as hard as possible to cope with changes on sonatype.org|com >:-(
I will try to deploy the new artifacts tomorrow.

**Comment by LoadingByte** [comment]
Oh my god, you are really quick! I'll be able to replace the giant blob that is POI with your lean library just in time for the release of [Cinecred](https://github.com/LoadingByte/cinecred) 1.6.0.

**Comment by rabanti-github** [comment]
Hi,
NanoXLSX4j is now available in version 2.4.0 (may be delayed due to replication in Maven Central).
It should now not pointing anymore to the parent POM. Also, the default column style was implemented.
Please let me know, if something is not working as excepted.

PS: Good to hear that the Java version may be useful too :-) .The .NET versions have almost its first quarter Million downloads. Let's see whether the Java version will also get some drive.

**Comment by LoadingByte** [comment]
Everything works exactly as expected. I'm really grateful for your super-fast support :)

Btw. I don't know why your library has so few users right now. The main competitors POI and fastexcel both require quite some dependencies, while yours is pretty lean. I hope it will get more traction.
